install_config/topics/configuring_for_vsphere_storage.adoc

```diff
@@ -0,0 +1,191 @@
+// Module included in the following assemblies:
+//
+// * install_config/configuring_for_vsphere.adoc
+
+[id='vsphere-configuring-storage_{context}']
+= Configuring {product-title} to use vSphere storage
+
+{product-title} supports VMware vSphere's Virtual Machine Disk (VMDK) volumes.
+You can provision your {product-title} cluster with
+xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage] using link:https://www.vmware.com/au/products/vsphere.html[VMware
+vSphere]. Some familiarity with Kubernetes and VMware vSphere is assumed.
+
+{product-title} creates the disk in vSphere and attaches the disk to the proper
+instance.
+
+
+The {product-title}
+xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+volume (PV)] framework allows administrators to provision a cluster with persistent
+storage and gives users a way to request those resources without having any
+knowledge of the underlying infrastructure. vSphere VMDK volumes can be
+xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[provisioned
+dynamically].
+
+PVs are not bound to a single project or namespace; they can be
+shared across the {product-title} cluster.
+xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[PV claims], however, are specific to a project or namespace and can be
+requested by users.
+
+[IMPORTANT]
+====
+High availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+
+[discrete]
+== Prerequisites
+
+Before creating PVs using vSphere, ensure your
+{product-title} cluster meets the following requirements:
+
+* {product-title} must first be for vSphere Cloud Provider
+* Each node host in the infrastructure must match the vSphere VM name.
+* Each node host must be in the same resource group.
+
+[IMPORTANT]
+====
```

Comment:
I would take this procedure out of the IMPORTANT box. If this is a prerequisite procedure, we can list it as such.

Reply:
@dav1x  can you provide information here if these steps are actually required or a way to meet the requirements from @ahardin-rh

Reply:
Actually, can we remove this line completely? It's a best practice but not required.